lms/djangoapps/discussion_api/api.py

```diff
@@ -274,6 +276,7 @@ def get_thread_list(
         "last_activity_at".
     order_direction: The direction in which to sort the threads by. The only
         values are "asc" or "desc". The default is "desc".
+    inline_discussion_id: ID of an inline discussion for any module
```

Comment:
Can we rename this to simply "discussion_id"?  For 2 reasons:
* To use a field name that is consistent with the discussion's `student_view_data` contract.
* In the future, we may have discussions in multiple places, and not necessarily only as inline.